sos/plugins/openstack_cinder.py

```diff
@@ -27,8 +27,10 @@ class OpenStackCinder(Plugin):
     plugin_name = "openstack_cinder"
     profiles = ('openstack', 'openstack_controller')
 
-    option_list = [("db", "gathers openstack cinder db version", "slow",
-                    False)]
+    option_list = [
+        ("db", "gathers openstack cinder db version", "slow", False),
+        ("exclude", "Ignore files matching this path spec", "", "")
+    ]
```

Comment:
NACK. We're not adding per-plugin options to selectively shoot arbitrary holes in a plugin's file lists (especially without leaving any trace behind - remember that this data is used principally for troubleshooting and problem solving).